snapshot-labs/snapshot
# feat(flagged spaces): conceal flagged spaces similar to flagged proposals

### Issues
#3860 

### Changes 
* create WarningHiddenContent element for reusability
* remove the `key` property on `router-view` because of unnecessary components reloading
* small code improvements

### How to test
1. Open flagged space
2. Ensure that its content is hidden behind a warning message
3. Do the same with flagged proposals

### To-Do
- [ ] confirm design and text for warning message

### Self-review checklist
